docs/src/pages/demos/selection-controls/CheckboxLabels.js

```diff
@@ -9,7 +9,7 @@ import CheckBoxIcon from '@material-ui/icons/CheckBox';
 import Favorite from '@material-ui/icons/Favorite';
 import FavoriteBorder from '@material-ui/icons/FavoriteBorder';
 
-const GreenCheckbox = withStyles({
+const styles = () => ({
```

Comment:
What happens when you only convert from JS? I thought type inference takes care of this?

Reply:
JS does, the formatting had changed it to equal the styles variable I created in the tsx file.  I refactored it to exclude introducing that styles variable.  I hope this is what you're looking for.  I plan on continuing to work on these so I'll improve my contributions hopefully.

Reply:
Changing the JS is only a measure of last resort (or if there was actually a bug). One goal of this effort is to see how good our types are. The less runtime you have to change the better.